content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -417,6 +417,86 @@ kubectl patch deployment patch-demo --patch-file patch-file.json
 kubectl patch deployment patch-demo --patch '{"spec": {"template": {"spec": {"containers": [{"name": "patch-demo-ctr-2","image": "redis"}]}}}}'
 ```
 
+### Update an object's replica count using `kubectl patch` with `--subresource` {#scale-kubectl-patch}
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+The flag `--subresource=[subresource-name]` is used with kubectl commands like get, patch, edit and replace to fetch and update `status` and `scale` subresources of the resources (applicable for kubectl version v1.24 or more). This flag is used with all the API resources (built-in and CRs) which has `status` or `scale` subresource. Deployment is one of the examples which supports these subresources.
+
+Here's a manifest for a Deployment that has two replicas:
+
+{{< codenew file="application/deployment-subresource.yaml" >}}
```

Comment:
Do we have to introduce a new YAML for this task?

Reply:
Hi @tengqm, thanks for taking look at this.

>Do we have to introduce a new YAML for this task?

For every example/strategy used [In Update API Objects in Place Using kubectl patch](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/) task, there is a separate deployment YAML and has different configurations according to strategy. So for this, we can use a new deployment YAML that is related to this only so it will be more useful to the users/readers. it maintains the uniformity of the docs.

Reply:
@Shubham82 I asked this question because I failed to find anything special in this `deployment-subresource.yaml` file. In fact, it can be any generic Deployment, right? This was not meant to be a blocker for this PR. My intent is to make sure that we are adding new example files for good reasons. We have [a test case for all examples](https://github.com/kubernetes/website/blob/main/content/en/examples/examples_test.go). I'm not convinced that a new file should be added for this scenario. Thoughts?

Reply:
Hi @tengqm
I agree with your point, it is a generic Deployment, but we can't use the deployment mentioned in the task. if the same (kind of) generic deployment is already used in the docs we can use that one also. could you please point me to that deployment I will test and update the content accordingly to that? WDYT?

Reply:
Here are some existing Deployment manifests:

- content/en/examples/application/deployment.yaml
- content/en/examples/application/deployment-scale.yaml
- content/en/examples/application/simple_deployment.yaml 
- content/en/examples/application/update_deployment.yaml 

They are pretty much the same thing repeated again and again.

Reply:
I have updated it. I used content/en/examples/application/deployment.yaml manifest for this particular task.
PTAL